bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/utils/MagentaTVLogger.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.magentatv.internal.utils;
+
+import java.text.MessageFormat;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link MagentaTVLogger} class implements a wrapper for slf4j.Logger and
+ * inserts a prefix to every messages, which makes it easier to read between all
+ * the other messages.
+ *
+ * @author Markus Michels - Initial contribution
+ */
+@NonNullByDefault
+public class MagentaTVLogger {
+    private Logger logger;
+
+    public MagentaTVLogger(Class<?> clazz, String module) {
+        logger = LoggerFactory.getLogger(clazz);
+    }
+
+    public void fatal(String message, Object... a) {
+        logger.warn("FATAL: {}", fmt(message, a));
+    }
+
+    public void info(String message, Object... a) {
+        logger.info("{}", fmt(message, a));
+    }
+
+    public void debug(String message, Object... a) {
+        logger.debug("{}", fmt(message, a));
+    }
+
+    public void trace(String message, Object... a) {
+        logger.trace("{}", fmt(message, a));
+    }
+
+    public void exception(Exception e, String message, Object... a) {
```

Comment:
https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
This whole class `MagentaTVLogger` doesn't seem to needed. It does wrap to format, while this is standard functionality with the logger and can be done directly with the logger. So my suggestion is to just use the logger directly and remove this class.